notebooks/climate-assessment.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
Which data is being used for this notebook?!

Reply:
Jarmo's test data from climate-assessment: I assume safe for public consumption?

Reply:
@jkikstra can you confirm?